Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/core/business/wrapper/Mappings.kt

```diff
@@ -124,21 +126,29 @@ internal fun String.toZonedDateTime(): ZonedDateTime {
 internal val ZonedDateTime.asExternalString: String
     get() = this.toString()
 
-internal val DccData<out DccV1.MetaData>.type: String
+internal val DccData<out DccV1.MetaData>.asExternalType: CertificateType
+    get() = when (certificate) {
+        is VaccinationDccV1 -> CertificateType.VACCINATION
+        is TestDccV1 -> CertificateType.TEST
+        is RecoveryDccV1 -> CertificateType.RECOVERY
+        else -> throw IllegalArgumentException("Unknown certificate type.")
```

Comment:
should we treat other types as `CertificateType.GENERAL` ?

Reply:
There is no CertificateType.GENERAL any more.